LongWarOfTheChosen/Src/LW_WeaponsAndArmor/Classes/X2Item_FactionWeapons.uc

```diff
@@ -762,7 +169,7 @@ static function X2DataTemplate CreateBullpup_Coil_Template()
 	Template.EquipSound = "Magnetic_Weapon_Equip";
 	Template.Tier = 4;
 
-	Template.RangeAccuracy = default.SKIRMISHER_COIL_RANGE;
+	Template.RangeAccuracy = default.SKIRMISHER_SMG_RANGE;
```

Comment:
Im general, isn't it a standard to have each weapon range per weapon, and not per weapon type?

Reply:
Seems like, although I don't know why. There's an awful lot of duplicate config. I don't even see a good reason to have different ranges for different weapon tiers. 🤷 For consistency, I'll split the Skirmisher SMG range into tiers and just do a lot of copying and pasting.

Reply:
Actually, I've changed my mind. I'd like to move to a single range config for all tiers of a weapon or range type. It will make updating range tables more reliable, since only one needs to be updated instead of five. There is always the option to add tier-specific range tables when we actually want a tier to have different ranges, but I'm just not sold on that being a good idea (except perhaps for plasma, which has its unique bonuses).